plugins/embeddedjsp/src/main/java/org/apache/struts2/jasper/compiler/JspReader.java

```diff
@@ -209,7 +209,6 @@ String getText(Mark start, Mark stop) throws JasperException {
         CharArrayWriter caw = new CharArrayWriter();
         while (!stop.equals(mark()))
             caw.write(nextChar());
-        caw.close();
```

Comment:
The API Docs and JDK source confirm that for `CharArrayWriter` instances, calls to `close()` are no-ops (no effect).
So I think this should be a safe change.